NEWS.rst

```diff
@@ -18,12 +18,17 @@ iterators of the alignments to reset as well, which is bug prone. Instead,
 calling ``iter`` on a ``PairwiseAlignments`` object will now return itself. The
 iterator can be reset by calling the ``rewind`` method.
 
+Calling `secondary_structure_fraction` on a ``ProtParam.ProteinAnalysis``
+object historically returned (sheet, turn, helix) while claiming to return
+(helix, turn, sheet). This was fixed to correctly return (helix, turn, sheet).
+
```

Comment:
Add something like "... and the amino acids considered was revised as per recent literature."?

Reply:
Additionally, the amino acids considered were revised as per recent literature.